README.md

```diff
@@ -100,7 +113,9 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema describes a data table within a database. Specifically, the schema should include column names, data type for each column, and constraints for each column.
+
+In a SQL database we can use a foreign key to create a one-to-many relationship. For example, if a person can have many wishes and each wish is unique to one person, then the wish would be attached to a foreign key that is the id of a specific person.
```

Comment:
Just to clarify, the schema describes the structure of the entire database. Included in that is the description of each table in the database. [I like this analogy](http://stackoverflow.com/a/298751)